apps/src/lib/node/ledger/shell/vote_extensions/bridge_pool_vext.rs

```diff
@@ -353,7 +359,36 @@ mod test_bp_vote_extensions {
         .expect("Test failed");
 
         // we advance forward to the next epoch
-        assert_eq!(shell.start_new_epoch().0, 1);
+        let consensus_set: Vec<WeightedValidator> =
+            read_consensus_validator_set_addresses_with_stake(
+                &shell.wl_storage,
+                Epoch::default(),
+            )
+            .unwrap()
+            .into_iter()
+            .collect();
+
+        let val1 = consensus_set[0].clone();
+        let pkh1 = get_pkh_from_address(
+            &shell.wl_storage,
+            &params,
+            val1.address.clone(),
+            Epoch::default(),
+        );
+        let votes = vec![VoteInfo {
+            validator: Some(namada::tendermint_proto::abci::Validator {
+                address: pkh1.clone(),
+                power: u128::try_from(val1.bonded_stake).expect("Test failed")
+                    as i64,
+            }),
+            signed_last_block: true,
+        }];
+        let req = FinalizeBlock {
+            proposer_address: pkh1,
+            votes,
+            ..Default::default()
+        };
+        assert_eq!(shell.start_new_epoch(Some(req)).0, 1);
```

Comment:
```rs
assert_eq!(shell.start_new_epoch().0, 1);
```
did this stop working? :\

Reply:
No it's just that we need a block proposer all the times from this pr, so I needed to construct a proper `FinalizeBlock` request